src/navigator_data_ingest/base/types.py

```diff
@@ -28,6 +38,30 @@ class DocumentType(str, Enum):
     LITIGATION = "Litigation"
 
 
+class UpdateTypes(str, Enum):
```

Comment:
@joel-wright - Guessing this defines the scope of what we are allowing to change on ingest?

Reply:
Those concepts don't really make their way into the pipeline, the pipeline is only really concerned with a single view of a document as "maybe something to parse & additional information to index". What is the purpose of defining different "types" of update?

Reply:
We strictly want to assert the type of update that has been made so that we can define which action to take. 

For example as there are duplicate fields in some tables - like id; we may in the future want to take different actions based upon which table the change has been made.

Reply:
@THOR300 ?

Reply:
https://github.com/climatepolicyradar/navigator-data-ingest/blob/feature/input-document-with-updates/src/navigator_data_ingest/base/updated_document_actions.py#L124

This is the function where the logic for which action to perform is identified. If you guys think that we don't need to consider the type here then I can simplify this.